README.md

```diff
@@ -89,10 +89,10 @@ The `distro` package implements a robust and inclusive way of retrieving the
 information about a distribution based on new standards and old methods,
 namely from these data sources (from high to low precedence):
 
-* The os-release file `/etc/os-release`, if present.
+* The os-release file `/etc/os-release` if present, with a fall-back on `/usr/lib/os-release` if needed.
 * The output of the `lsb_release` command, if available.
 * The distro release file (`/etc/*(-|_)(release|version)`), if present.
-* The `uname` command for BSD based distrubtions.
+* The `uname` command for BSD based distributions.
```

Comment:
With my picky hat on: I'm all for fixing typos but I'd vote to do that in a dedicated commit so that people after you and me will be able to look at pure semantic changes without skip-reading over spelling fixing. Just my 2 cents.

Reply:
I understand your point.
As I didn't think, back in April, to _take over_ (?) this project, I wanted to propose many improvements as possible. If I do, I'll propose them again in a dedicated PR, with dedicated commits 👌